tests/Integration/Database/EloquentModelTest.php

```diff
@@ -69,6 +71,54 @@ public function testAttributeChanges()
         $this->assertTrue($user->wasChanged());
         $this->assertTrue($user->wasChanged('name'));
     }
+
+    public function testAttributeTransitionedFromTo()
```

Comment:
The following test uses only `transitioningTo ` ;  shouldn't we have some use cases for `transitionedTo` ?